met-web/src/components/landing/TileBlock.tsx

```diff
@@ -54,63 +58,47 @@ const TileBlock = () => {
                 xs={10}
             >
                 <NoResult />
+                <ul tabIndex={0} aria-label="Engagements list. No results."></ul>
```

Comment:
Text elements should generally not have tabIndex set on them - screen readers can navigate text on their own

Reply:
https://webaim.org/techniques/keyboard/tabindex#zero-negative-one
https://web.dev/articles/control-focus-with-tabindex
https://www.a11y-collective.com/blog/tabindex-accessibility/#h-using-tabindex-value-zero-on-non-interactive-elements